src/app/services/auth.service.ts

```diff
@@ -1,16 +1,19 @@
 import { Injectable } from '@angular/core';
 import { AngularFireAuth } from '@angular/fire/auth';
-import { FirebaseService } from './firebase.service';
 import { switchMap, map, shareReplay } from 'rxjs/operators';
 import { from, of, Observable } from 'rxjs';
 
+// Make sure that firebase/auth is included in the main bundle
+// (not lazy-loaded).
```

Comment:
```suggestion
// (not lazy-loaded, because relying on it to lazy-load was not working). 
```

Reply:
Oh, fair ☺️ Sorry, my understanding of the issue is also shaky, so I didn't write a very detailed comment :)